packages/airnode-validator/package.json

```diff
@@ -3,19 +3,24 @@
   "license": "MIT",
   "version": "0.6.0",
   "private": false,
-  "main": "./dist/src/index.js",
-  "types": "./dist/src/index",
```

Comment:
I had to remove the "types" because it wasn't working. I've checked that the typings are working in a test project.